epub33/rs/index.html

```diff
@@ -384,14 +384,18 @@ <h3>Network access</h3>
 			<section id="sec-epub-rs-external-links">
 				<h3>External links</h3>
 
-				<p data-tests="#pub-external-links">Reading systems SHOULD open links that resolve outside the <a>EPUB
-						publication</a> in a new browser instance to ensure that the browser's security and privacy
-					controls are available to users.</p>
+				<p>Reading systems SHOULD alert users before opening hyperlinks that declare a <a
+						data-cite="url#concept-url-scheme">scheme</a> [[url]] (i.e., that resolve outside the [=EPUB
+					publication=] or require a specialized application to open).</p>
+
+				<p data-tests="#pub-external-links">Reading systems SHOULD open links that declare a scheme in an
+					appropriate application to ensure security and privacy controls are available.</p>
 
 				<p>Although links to external web sites and resources are commonly found in <a>EPUB content
-						documents</a>, these are not the only sources. For example, if a reading system provides access
-					to <a data-cite="epub-33#sec-link-elem">linked records</a> [[epub-33]] in the <a>package
-						document</a> metadata, it should similarly open the links in a new browser instance.</p>
+						documents</a>, these are not the only sources of hyperlinks with schemes. For example, if a
+					reading system provides access to external <a data-cite="epub-33#sec-link-elem">linked records</a>
+					[[epub-33]] in the <a>package document</a> metadata, it should open the links in a new browser
+					instance.</p>
```

Comment:
For the sake of consistency, we should use the term "specialized external application" (or "user agent") here as well.

Reply:
Actually, I'm not sure there's any point to this paragraph anymore. It was added because the first one was so focused on opening web content. At most we might mention that links aren't only found in content documents, but I'm not sure that's really a major clarification.